Makefile

```diff
@@ -61,8 +61,6 @@ check: test
 test: verifiers build
 	@echo "Running unit tests"
 	@go test $(GOFLAGS) -tags kqueue ./...
-	@echo "Verifying build"
-	@(env bash $(PWD)/buildscripts/verify-build.sh)
```

Comment:
Move it as separate build target for manual usage.

Reply:
The reason to avoid it was to ensure that make is not involved in this test. I have no idea right now why the script itself fails to execute.

Reply:
It looks like the failure was travis bug.  I think we should remove it from travis if it fails with valid reasons.

Reply:
At least it might run as a script perhaps since the test on the PR itself runs fine.

Reply:
I moved it as a separate target for now.